src/Lykke.Job.OpsGenie.Services/OpsGenieApi/Contracts/CreateAlertResponceContract.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Newtonsoft.Json;
+
+namespace Lykke.Job.OpsGenie.Services.OpsGenieApi.Contracts
+{
+    internal class CreateAlertResponceContract
+    {
+        [JsonProperty("requestId")]
+        public string RequestId { get; set; }
+
+        [JsonProperty("took")]
+        public double Took { get; set; }
+
+        [JsonProperty("Result")]
```

Comment:
Rest of field names are started from lower case. And do we really need all these JsonProperty? looks like they just match property names